stellargraph/layer/deep_graph_infomax.py

```diff
@@ -145,12 +154,17 @@ def embedding_model(self, model):
         Returns:
             input and output layers for use with a keras model
         """
-        x_emb_in = model.inputs[len(self._corruptible_inputs_idxs) :]
 
+        if not any(layer.name == self._unique_id for layer in model.layers):
+            raise ValueError(
+                f"model: model must be a keras model with inputs and outputs created "
+                f"by this instance of DeepGraphInfoMax.build() function."
```

Comment:
It's not entirely clear to me what this sentence means, because I parse it as "by this instance of (DeepGraphInfoMax.build() function)". I guess you were trying to go for "by (this instance of DeepGraphInfoMax)'s .build() function"? Maybe:

```suggestion
                f"by the build() method of this instance of DeepGraphInfoMax"
```

Reply:
nice, that reads easier :+1: